src/unix/linux_like/linux/align.rs

```diff
@@ -63,6 +63,13 @@ macro_rules! expand_align {
                 pub fd: ::c_int,
                 pub pid: ::c_int,
             }
+
+            #[repr(align(2))]
+            pub struct fanotify_event_info_header {
```

Comment:
https://github.com/torvalds/linux/blob/0c3836482481200ead7b416ca80c68a29cfdaabd/include/uapi/linux/fanotify.h#L153 doesn't seem to have an alignment attribute, and I think the default alignment should naturally be 16 bits. Why is it needed here?

Reply:
Also I think this would cause a problem building on ancient versions with alignment since this doesn't have a fallback